core/jobs/batch_jobs/test_gcs_io_job_test.py

```diff
@@ -22,42 +22,31 @@
 from core.jobs.batch_jobs import test_gcs_io_job
 from core.jobs.types import job_run_result
 from core.platform import models
+from core.platform.storage import cloud_storage_emulator
```

Comment:
No, do not import this. Or if it is absolutely needed do it inside `if MYPY`

Reply:
Removed, was required when we were returning the blob.